homeassistant/components/nut/device_action.py

```diff
@@ -30,23 +30,20 @@ async def async_get_actions(
     hass: HomeAssistant, device_id: str
 ) -> list[dict[str, str]]:
     """List device actions for Network UPS Tools (NUT) devices."""
-    if (entry_id := _get_entry_id_from_device_id(hass, device_id)) is None:
-        return []
-
-    actions = []
-    supported_commands: set[str] = hass.data[DOMAIN][entry_id][
-        DEVICE_SUPPORTED_COMMANDS
-    ]
-    base_action = {
-        CONF_DEVICE_ID: device_id,
-        CONF_DOMAIN: DOMAIN,
-    }
-    for command_name in supported_commands:
-        device_action_name = _get_device_action_name(command_name)
-
-        actions.append({**base_action, CONF_TYPE: device_action_name})
-
-    return actions
+    return (
```

Comment:
This one didn't turn out how I was envisioning.  I'll put up a suggestion shortly 👍

Reply:
You were faster than me. I meant to reply saying that I could have gone too far on making the code more succinct, at the expense of code readability. Your suggestion fixes that. Thanks!